### PR TITLE
Refactor challenges

### PR DESCRIPTION
**Pull request by ioboi**
## Points of interest:

* New layout
* Use sections for challenges (separated files)
* Badges make it easier to recognize the required skill level

## Improvements
I tried to correct some texts in the translations but it is important that we will work a bit more on the content. There are some inconsistencies that have to be removed in a later pull request.

### CODE REVIEW

File: content/challenges/arduipi-sandbox.de.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "ArduiPi-Sandbox"
+levels:
+    - Beginer
+    - Intermediate
+    - Advanced
+hardware:
+    - SF-PiCar-V
+    - Sensoren nach deiner Wahl
```
Comment (DonGiovanni83) [inline content/challenges/arduipi-sandbox.de.md]: Should be  "_Sensoren deiner Wahl_" like in every other challenge.

File: content/challenges/sf-pi-car-v.en.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: "SF-PICar-V Computer Vision with SimpleCV"
+levels:
+    - Intermediate
+    - Advanced
+hardware:
+    - SF-PiCar-V
+    - Sensors of your choice
```
Comment (DonGiovanni83) [inline content/challenges/sf-pi-car-v.en.md]: Should be  "_Sensores of choice_" like in every other challenge.

Comment (ioboi) [reply content/challenges/sf-pi-car-v.en.md]: I will change every english challenge to "Sensors of your choice" as it is more directed towards potential attendees.

**Comment by ioboi** [comment]
Fixed inconsistencies.

**Pull request merged**